observatory-platform/observatory/platform/terraform/api/main.tf

```
@@ -0,0 +1,163 @@
+########################################################################################################################
+# Enable API services
+########################################################################################################################
+
+resource "google_project_service" "servicemanagement" {
+  project = var.google_cloud.project_id
+  service = "servicemanagement.googleapis.com"
+  disable_on_destroy = false
+}
+
+resource "google_project_service" "servicecontrol" {
+  project = var.google_cloud.project_id
+  service = "servicecontrol.googleapis.com"
+  disable_on_destroy = false
+}
+
+resource "google_project_service" "endpoints" {
+  project = var.google_cloud.project_id
+  service = "endpoints.googleapis.com"
+  disable_on_destroy = false
+}
+
+# Enable service
+resource "google_project_service" "api-project-service" {
+  service = google_endpoints_service.api.service_name
+  project = var.google_cloud.project_id
+  depends_on = [google_endpoints_service.api]
+  disable_on_destroy = false
+}
+
+########################################################################################################################
+# Cloud Run backend for API
+########################################################################################################################
+resource "google_service_account" "api-backend_service_account" {
+  account_id   = "api-backend"
+  display_name = "API backend Service Account"
+  description = "The Google Service Account used by the cloud run backend"
+}
+
+# Create elasticsearch secrets
+module "elasticsearch-logins" {
+  for_each = var.elasticsearch
+  source = "../secret"
+  secret_id = "elasticsearch-${each.key}"
+  secret_data = each.value
+  service_account_email = google_service_account.api-backend_service_account.email
+}
+
+resource "google_cloud_run_service" "backend" {
```

```suggestion
resource "google_cloud_run_service" "api_backend" {
```

It would be good to make the naming of this consistent with the other cloud run service.